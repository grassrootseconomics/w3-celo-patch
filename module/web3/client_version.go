package web3

import (
	"github.com/grassrootseconomics/w3-celo-patch/internal/module"
	"github.com/grassrootseconomics/w3-celo-patch/w3types"
)

// ClientVersion requests the endpoints client version.
func ClientVersion() w3types.CallerFactory[string] {
	return module.NewFactory[string](
		"web3_clientVersion",
		nil,
	)
}
