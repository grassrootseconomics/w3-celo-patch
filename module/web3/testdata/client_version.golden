> {"jsonrpc":"2.0","id":1,"method":"web3_clientVersion"}
< {"jsonrpc":"2.0","id":1,"result":"Geth"}
